tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3187,6 +3189,13 @@ declare namespace ts {
             private delayUpdateSourceInfoProjects;
             private delayUpdateProjectsOfScriptInfoPath;
             private handleDeletedFile;
+            /**
+             * This function goes through all the openFiles and tries to file the config file for them.
+             * If the config file is found and it refers to existing project, it schedules the reload it for reload
+             * If there is no existing project it just opens the configured project for the config file
+             * shouldReloadProjectFor provides a way to filter out files to reload configured project for
+             */
+            private delayReloadConfiguredProjectsForFile;
```

Comment:
Not for this PR of course, but it sure seems like we should be dropping JSDoc on private props in dts emit...

Reply:
https://github.com/microsoft/TypeScript/issues/58145